### ISSUE TITLE
HOPPERS Bugged

### ISSUE THREAD

**Issue by maikerumine** [Wuzzy2/MineClone2-Bugs#365]
`2017-08-08 17:08:01: ERROR[Main]: ServerError: AsyncErr: ServerThread::run Lua: Runtime error from mod 'mcl_hoppers' in callback LuaABM::trigger(): Invalid position (expected table got nil).
2017-08-08 17:08:01: ERROR[Main]: stack traceback:
2017-08-08 17:08:01: ERROR[Main]: 	[C]: in function 'get_meta'
2017-08-08 17:08:01: ERROR[Main]: 	D:\MTSERVER-TEST\bin\..\mods\mc2plus\mcl_util\init.lua:142: in function 'move_item_container'
2017-08-08 17:08:01: ERROR[Main]: 	...\bin\..\games\MineClone2\mods\ITEMS\mcl_hoppers\init.lua:268: in function <...\bin\..\games\MineClone2\mods\ITEMS\mcl_hoppers\init.lua:259>
2017-08-08 17:08:01: ERROR[Main]: stack traceback:
2017-08-08 17:08:01: ERROR[Main]: 	[C]: in function 'get_meta'
2017-08-08 17:08:01: ERROR[Main]: 	D:\MTSERVER-TEST\bin\..\mods\mc2plus\mcl_util\init.lua:142: in function 'move_item_container'
2017-08-08 17:08:01: ERROR[Main]: 	...\bin\..\games\MineClone2\mods\ITEMS\mcl_hoppers\init.lua:268: in function <...\bin\..\games\MineClone2\mods\ITEMS\mcl_hoppers\init.lua:259>`

and then:

`2017-08-08 17:08:42: ERROR[Main]: ServerError: AsyncErr: ServerThread::run Lua: Runtime error from mod 'mcl_hoppers' in callback LuaABM::trigger(): Invalid position (expected table got nil).
2017-08-08 17:08:42: ERROR[Main]: stack traceback:
2017-08-08 17:08:42: ERROR[Main]: 	[C]: in function 'get_meta'
2017-08-08 17:08:42: ERROR[Main]: 	D:\MTSERVER-TEST\bin\..\mods\mc2plus\mcl_util\init.lua:142: in function 'move_item_container'
2017-08-08 17:08:42: ERROR[Main]: 	...\bin\..\games\MineClone2\mods\ITEMS\mcl_hoppers\init.lua:268: in function <...\bin\..\games\MineClone2\mods\ITEMS\mcl_hoppers\init.lua:259>
2017-08-08 17:08:42: ERROR[Main]: stack traceback:
2017-08-08 17:08:42: ERROR[Main]: 	[C]: in function 'get_meta'
2017-08-08 17:08:42: ERROR[Main]: 	D:\MTSERVER-TEST\bin\..\mods\mc2plus\mcl_util\init.lua:142: in function 'move_item_container'
2017-08-08 17:08:42: ERROR[Main]: 	...\bin\..\games\MineClone2\mods\ITEMS\mcl_hoppers\init.lua:268: in function <...\bin\..\games\MineClone2\mods\ITEMS\mcl_hoppers\init.lua:259>`

**Comment by maikerumine** [comment]
This is because mobs mc items were lost.  :/

**Comment by Wuzzy2** [comment]
Your post in readable form: 
```
2017-08-08 17:08:01: ERROR[Main]: ServerError: AsyncErr: ServerThread::run Lua: Runtime error from mod 'mcl_hoppers' in callback LuaABM::trigger(): Invalid position (expected table got nil).
2017-08-08 17:08:01: ERROR[Main]: stack traceback:
2017-08-08 17:08:01: ERROR[Main]: 	[C]: in function 'get_meta'
2017-08-08 17:08:01: ERROR[Main]: 	D:\MTSERVER-TEST\bin\..\mods\mc2plus\mcl_util\init.lua:142: in function 'move_item_container'
2017-08-08 17:08:01: ERROR[Main]: 	...\bin\..\games\MineClone2\mods\ITEMS\mcl_hoppers\init.lua:268: in function <...\bin\..\games\MineClone2\mods\ITEMS\mcl_hoppers\init.lua:259>
2017-08-08 17:08:01: ERROR[Main]: stack traceback:
2017-08-08 17:08:01: ERROR[Main]: 	[C]: in function 'get_meta'
2017-08-08 17:08:01: ERROR[Main]: 	D:\MTSERVER-TEST\bin\..\mods\mc2plus\mcl_util\init.lua:142: in function 'move_item_container'
2017-08-08 17:08:01: ERROR[Main]: 	...\bin\..\games\MineClone2\mods\ITEMS\mcl_hoppers\init.lua:268: in function <...\bin\..\games\MineClone2\mods\ITEMS\mcl_hoppers\init.lua:259>
```

```
2017-08-08 17:08:42: ERROR[Main]: ServerError: AsyncErr: ServerThread::run Lua: Runtime error from mod 'mcl_hoppers' in callback LuaABM::trigger(): Invalid position (expected table got nil).
2017-08-08 17:08:42: ERROR[Main]: stack traceback:
2017-08-08 17:08:42: ERROR[Main]: 	[C]: in function 'get_meta'
2017-08-08 17:08:42: ERROR[Main]: 	D:\MTSERVER-TEST\bin\..\mods\mc2plus\mcl_util\init.lua:142: in function 'move_item_container'
2017-08-08 17:08:42: ERROR[Main]: 	...\bin\..\games\MineClone2\mods\ITEMS\mcl_hoppers\init.lua:268: in function <...\bin\..\games\MineClone2\mods\ITEMS\mcl_hoppers\init.lua:259>
2017-08-08 17:08:42: ERROR[Main]: stack traceback:
2017-08-08 17:08:42: ERROR[Main]: 	[C]: in function 'get_meta'
2017-08-08 17:08:42: ERROR[Main]: 	D:\MTSERVER-TEST\bin\..\mods\mc2plus\mcl_util\init.lua:142: in function 'move_item_container'
2017-08-08 17:08:42: ERROR[Main]: 	...\bin\..\games\MineClone2\mods\ITEMS\mcl_hoppers\init.lua:268: in function <...\bin\..\games\MineClone2\mods\ITEMS\mcl_hoppers\init.lua:259>
```

**Comment by Wuzzy2** [comment]
You should write down how to reproduce this crash.
List the exact steps you did before this happened.


Also list all mods and the MCL2 version number.

**Comment by Wuzzy2** [comment]
@maikerumine: Your issue is almot certainly invalid.
I think your problem is that you used `mcl_util` from `mc2plus`.
I have updated `mcl_util` in MCL2 quite heavily, so your fork of `mcl_util` is outdated.
Just disable the mod for your world, or update your fork and it should work again.

As I already stated in the forums, we should definitely find a way on how to get rid of your `mcl_util` fork altogether so you are safe from updating hell.
